code/modules/power/fusion/fuel_assembly/fuel_compressor.dm

```diff
@@ -63,7 +63,7 @@
 					blitzprogress = 1
 					verbs |= /obj/machinery/fusion_fuel_compressor/verb/eject_sheet
 					return
-				to_chat(user, "<span class='warning'>You need at least 25 [mat.sheet_plural_name] to make a fuel rod.</span>")
+				to_chat(user, "<span class='warning'>You need at least 15 [mat.sheet_plural_name] to make a fuel rod.</span>")
```

Comment:
Also this may as well use the macro for how many is needed since the problem in the first place was that the macro was changed but the text was hardcoded :v